addons/l10n_cl/report/invoice_report.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models
+
+
+class ReportInvoice(models.AbstractModel):
+    _inherit = 'report.account.report_invoice'
+
+    def _formatted_vat(self, vat):
+        n_vat = vat.split('-')[0]
+        n_dv = vat.split('-')[1]
+        return '%s-%s' % ('{:,.0f}'.format(int(n_vat)).replace(',', '.'), n_dv)
+
+    @api.model
+    def _get_report_values(self, docids, data=None):
+        rslt = super()._get_report_values(docids, data)
+        rslt['_formatted_vat'] = self._formatted_vat
```

Comment:
was not there another place where we could add these values?  It could even just be a method on the invoice.  It is a bit a lot of diff for just passing a method...

Reply:
ok.. doing it